rom/dev/README.md

```diff
@@ -62,7 +62,7 @@ Following are the main FUSE & Architectural Registers used by the Caliptra ROM f
 | FUSE_LMS_REVOCATION             | 32           | Manufacturer LMS Public Key Revocation Mask             |
 | FUSE_MLDSA_REVOCATION           | 32           | Manufacturer MLDSA Public Key Revocation Mask           |
 | FUSE_OWNER_PK_HASH              | 384          | Owner ECC and LMS or MLDSA Public Key Hash              |
-| FUSE_FMC_KEY_MANIFEST_SVN       | 32           | FMC Security Version Number                             |
+| FUSE_FMC_KEY_MANIFEST_SVN       | 32           | FMC Security Version Number (Deprecated)                |
```

Comment:
nit: Do we want to reclaim these fuses for something else in 2.0? If so, we may want to mark it as reserved or something.

Reply:
Should I just delete the row from here?

Reply:
Ya I think that makes sense

Reply:
Done.